src/pages/index.tsx

```diff
@@ -1,99 +1,111 @@
-import {useWalletMultiButton} from "@solana/wallet-adapter-base-ui";
-import {useWalletModal} from "@solana/wallet-adapter-react-ui";
-import GetStarted from "@src/components/icons/GetStarted.icon";
-import {ProfileContext} from "@src/provider/ProfileProvider";
-import {api} from "@src/utils/api";
-import {LABELS, Routes} from "@src/utils/constants/constants";
-import {useRouter} from "next/router";
-import React, {useContext, useEffect, useMemo} from "react";
-import {toast} from "sonner";
+"use client"
```

Comment:
If i remove them, it will disturb the codebase.

Reply:
If I remove the lines of code you asked me to remove, it will break the codebase. Please let me know if you would still like me to remove them or help explain further.

Reply:
we're not using pages directory so we don't need any "use client" declaration.

Reply:
ok it was because I built the project separately first, I have remove all "use client"